build.rs

```diff
@@ -221,6 +221,7 @@ const DATA: &'static [(&'static str, &'static [&'static str], &'static [&'static
     ("msaatext", &[], &[]),
     ("mscat", &["guiddef", "minwindef", "mssip", "wincrypt", "winnt"], &[]),
     ("mschapp", &["basetsd", "minwindef", "winnt"], &["advapi32"]),
+    ("msdelta", &["basetsd", "minwindef", "wincrypt", "winnt"], &[]),
```

Comment:
You have to add `"msdelta"` to the third field here since that is the name of the import library that contains the symbols you are trying to link to.

Reply:
Thanks, it worked 🎉